experimental/juvix/contract.jvx

```diff
@@ -0,0 +1,198 @@
+Map = Data.Map
+
+-- common module interfaces for data structures in Common?
+MonoidMeasure : Common.Measure
+MonoidMeasure =
+  type T = Nat
+
+  -- (<>) is normal semi-group  type class function
```

Comment:
Are we assuming that the semi-group typeclass is automatically in scope? Should we `import Prelude` ?

Reply:
I assume we have implicit prelude?

Reply:
Hmm alright.